### PR TITLE
More integration tests

### PR DESCRIPTION
**Pull request by gabrielittner**
- integration tests for Kotlin project (main addition here is the assertion that the dokka `task` runs)
- check actual pom content
- introduce `fixtures/common` which will be copied for every tests but can be overridden by the test specific fixtures
- move build files to the fixtures because they are diverging more

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,13 +8,13 @@ POM_PACKAGING=jar
...
-POM_URL=http://github.com/vanniktech/gradle-maven-publish-plugin/
-POM_SCM_URL=http://github.com/vanniktech/gradle-maven-publish-plugin/
+POM_URL=https://github.com/vanniktech/gradle-maven-publish-plugin/
+POM_SCM_URL=https://github.com/vanniktech/gradle-maven-publish-plugin/
 POM_SCM_CONNECTION=scm:git:git://github.com/vanniktech/gradle-maven-publish-plugin.git
 POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/vanniktech/gradle-maven-publish-plugin.git
 
 POM_LICENCE_NAME=The Apache Software License, Version 2.0
-POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
+POM_LICENCE_URL=https://www.apache.org/licenses/LICENSE-2.0.txt
```
Comment (vanniktech) [inline gradle.properties]: Good one!

File: src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt [hunk 2]
```diff
@@ -38,48 +41,20 @@ class MavenPublishPluginIntegrationTest(
...
-            uploadArchives {
-              releaseRepositoryUrl = "file://${repoFolder.absolutePath}"
-              signing = true
-            }
-          }
-        }
-        """)
 
-    testProjectDir.newFile("gradle.properties").writeText("""
+    File("$FIXTURES/common").listFiles()?.forEach { it.copyRecursively(File(testProjectDir.root, it.name)) }
```
Comment (vanniktech) [inline src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt]: super nit: we could `testProjectDir.root.resolve(it.name)` feel free to do in another PR

File: src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt [hunk 3]
```diff
@@ -38,48 +41,20 @@ class MavenPublishPluginIntegrationTest(
...
+    File(testProjectDir.root, "gradle.properties").appendText("""
```
Comment (vanniktech) [inline src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt]: same here

File: src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt (hunk 2 above)
Comment (gabrielittner) [reply src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt]: I didn't know about resolve, nice 
43031c2c35292e7f1b67dbba167411acd9f8e0d1

File: src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt (hunk 3 above)
Comment (gabrielittner) [reply src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt]: 43031c2

**Pull request merged**